projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -226,12 +479,13 @@ export function addItemCommentAndWait(item: string, text: string) {
 }
 
 /**
- * checks whether the given header comment with item link is displayed on the given position
- * @param index index of the comment containing the link within the comment section
- * @param item name of the item
- * @param text text to be displayed
+ * Checks whether the given header comment with item link is displayed on the given position
+ * @param index Index of the comment containing the link within the comment section
+ * @param item Name of the item
+ * @param text Text to be displayed
  */
 export function checkItemComment(index: number, item: string, text: string) {
+  log(checkItemComment.name);
```

Comment:
I would suggest not to log the name of the method but to log the method description, for instance:
> Verifies an item comment

Reply:
done